src/printer.js

```diff
@@ -2342,6 +2354,7 @@ function typeIsFunction(type) {
 
 function printExportDeclaration(path, options, print) {
   const decl = path.getValue();
+  const semi =  options.noSemi ? "" : ";"
```

Comment:
yeah, looks like the codebase is overdue for a run of `npm run format:all`, I see loads of changes (not all of them look great? what's with the removal of all the newlines?)